### PR TITLE
update pricing docs

### PR DESCRIPTION
**Pull request by ebrodymoore**
## Test plan

Markdown rendering

## App preview:

- [Web](https://sg-web-ebm-pricing-update.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-uhirljnbsc.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

File: doc/admin/pricing.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Pricing
+
+Visit the [pricing page](https://about.sourcegraph.com/pricing) for up-to-date pricing. 
+
+## How are active users calculated?
+
+Any user who accesses Sourcegraph in a given month is considered a monthly active user (MAU). This includes but is not limited to:
```
[the remark this answers is not in the document]
Comment (ebrodymoore) [reply doc/admin/pricing.md:7]: Not logging in (and this listed as one of the exceptions in the second section) but if the login is successful you'll land on a page that is most likely logged

**Review by abschu — APPROVED** [review]

**Review by erzhtor — APPROVED** [review]
LGTM!

File: doc/admin/pricing.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- Conducting a search in the Sourcegraph UI or extension
```
Comment (sqs) [inline doc/admin/pricing.md:8]: @ebrodymoore looks like it needs a blank line before the `-` bullet list: https://docs.sourcegraph.com/admin/pricing#pricing

Comment (ebrodymoore) [reply doc/admin/pricing.md:8]: on it

**Pull request merged**